UI/window-basic-main-profiles.cpp

```diff
@@ -314,13 +315,11 @@ void OBSBasic::RefreshProfileCache()
 			continue;
 		}
 
-		std::string profileCandidate;
-		profileCandidate.reserve(entry.path().u8string().size() + OBSProfileSettingsFile.size() + 1);
-		profileCandidate.append(entry.path().u8string()).append("/").append(OBSProfileSettingsFile);
+		auto profileCandidate = entry.path() / profileSettingsFile;
```

Comment:
Shouldn't this be `const` as it's not mutated anywhere?

Reply:
It could be constified, yeah. It could probably also be `std::move`d in the construction of `OBSProfile`.